src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAXIMUM_NUMBER = 100;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
And make this object final.
Because as random object, it perfoms defined range of tasks (getting a color).
It has not an internal state which you will change. So from this point of view, this object is immutable.

(For instance, Person object can store surname as String. It won't be final, since we can change the surname. In case of colorSupplier, we do not want to do this. Also, final keyword can say java compiler to optimize some things under the hood)